tests/test_transforms/test_decomposition/test_detrend_transform.py

```diff
@@ -1,40 +1,87 @@
+import numpy as np
```

Comment:
It's not best practice to rewrite existing tests in case of incremental changes. 
Current task doesn't suggest breacking changes.

It's better to add new test cases if necessary.
If some tests are important. We should delete them in the next task, not in the same.

Reply:
The tests were very strange: they don't test that trend becomes around zero, they tested that mean of values become close to zero. This check can be somewhat useful for a plain trend, but it checks nothing for a polynomial trend (second order does nothing in this case). And this means that we should write new set of tests for `poly_degree` > 1. I think that it creates only confusion.